src/XR/features/WebXRControllerPointerSelection.ts

```diff
@@ -384,18 +387,23 @@ export class WebXRControllerPointerSelection extends WebXRAbstractFeature {
                             let zAxisMultiplier = this._scene.useRightHandedSystem? 1 : -1;
                             const indexTipPos = indexTipPose.transform.position;
                             const indexTipOrientation = indexTipPose.transform.orientation;
+                            controllerData.indexTipQuaternion.set(indexTipOrientation.x, indexTipOrientation.y, (indexTipOrientation.z * zAxisMultiplier), (indexTipOrientation.w * zAxisMultiplier));

```

Comment:
nit: should probably rename zAxisMultiplier to better reflect that it's related to right-hand coordinate systems, since it's used for more than just the z axis

Reply:
i  did rename this to zaxisRHSmultiplier, do you want it to be renamed something else?